tuf/ngclient/updater.py

```diff
@@ -360,7 +360,7 @@ def _load_snapshot(self) -> None:
                 version = snapshot_meta.version
 
             data = self._download_metadata("snapshot", length, version)
-            self._trusted_set.update_snapshot(data)
+            self._trusted_set.update_snapshot(data, trusted=False)
```

Comment:
```suggestion
            self._trusted_set.update_snapshot(data)
```

Nit: we defined a default, should we just use it? Is being explicit here a style recommendation?

Reply:
Even though we have a default value I prefer if we are explicit about this.
That way it's easier to distinguish when we consider the data as `trusted` and when we do not.
In this case, I see the default value more as a defense mechanism that `trusted` will be set.

Reply:
Not requiring any change here. But if we want to force ourselves to be explicit about how we call this internal API, why have the default?

Reply:
What do you think @jku? Should we have a default value for `trusted`?

Reply:
I don't care very much, but defining a default in internal API and then explicitly avoiding using that default as defense mechanism seems weird. So I'd say use the default or don't define it in the first place... (but it's really not a required change)

Reply:
I checked and if I decide to change the code to mandate `trusted` it will have too many other unrelated changes, so the tests could pass. It doesn't worth it.
I the argument from this function call as Joshua suggested.